.github/workflows/review-code.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
             file_content=$(git show "${{ github.event.pull_request.head.sha }}:$file" | jq -Rsa .)
 
             # Send the diff and file content to the backend for each file separately
-            curl -X POST "https://app5-latest.onrender.com/openai/api/v2/strategy/4/review-pr-diff" \
+            curl -X POST "https://app5-latest.onrender.com/ai/api/v1/strategy/1/review-pr-diff" \
```

Comment:
The use of a hardcoded URL can lead to security vulnerabilities and makes the code less maintainable. If this is part of a larger script or automation process, consider using a more secure method to handle URLs and sensitive data. (Severity: high)